core/resources/lights_frag.glsl

```diff
@@ -0,0 +1,39 @@
+varying vec4 v_amb;
+varying vec4 v_diff;
+varying vec4 v_spec;
+
+vec4 calculateLighting(in vec3 _eyeToPoint, in vec3 _normal) {
+	vec3 eye = vec3(0.0, 0.0, 1.0);
+
+	vec4 amb = v_amb;
+	vec4 diff = v_diff;
+	vec4 spec = v_spec;
+
+//  COMPUTE ALL LIGHTS
+#pragma tangram: lights_to_compute
```

Comment:
We should discuss how to simplify any dependencies between various injected fragments, e.g. avoid injections that themselves have injections in them. It's not yet clear that we want to support this level of complexity. To date I've kept the JS-based injection to simple single injection points mapped to key/value string pairs. cc @blair1618